perception/sub8_pointcloud/src/Classification.cpp

```diff
@@ -3,8 +3,8 @@ Classification::Classification(ros::NodeHandle *nh)
 {
   nh_ = nh;
   // pubTest = nh_->advertise<visualization_msgs::MarkerArray>( "/test_pub", 0 );
-  pubTest = nh_->advertise<pcl::PointCloud<pcl::PointXYZI>>("ogridgen/point_cloud_filtered", 0);
-  // pubTest2 = nh_->advertise<pcl::PointCloud<pcl::Normal>>("test_pub/normals", 0);
+  // pubTest = nh_->advertise<pcl::PointCloud<pcl::PointXYZI>>("ogridgen/point_cloud_filtered", 0);
```

Comment:
Dead code, leave a TODO comment if you want to remember to do stuff in the future